### PR TITLE
Ft contact

### PR DESCRIPTION
**Pull request by adisamicheal**
Contact us page

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1,2 @@
-VUE_APP_API_URL=''
+VUE_APP_API_URL='https://uptima.herokuapp.com'
```
Comment (addempsea) [inline .env:1]: add .env to the git ignore file

File: src/store/modules/contact.us.js [hunk 2]
```diff
@@ -1,15 +1,52 @@
...
+                        'updateSnackbar',
+                        {
+                            color: '#00C853',
+                            text: 'Thanks for reaching out to us!',
+                            showSnackbar: true
+                        },
+                        { root: true }
+                    );
+                    router.push('/');
+                }
```
Comment (addempsea) [inline src/store/modules/contact.us.js:27]: while it pushes to home, the snackbar still shows, right?

File: src/utils/api.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import instance from './axios';
+
+export default {
+    get(url) {
+        return instance.get(url);
+    },
```
Comment (addempsea) [inline src/utils/api.js:6]: try to format this, to be consistent across the code base

**Review by addempsea — COMMENTED** [review]
update me when you make the changes

File: src/store/modules/contact.us.js (hunk 2 above)
Comment (adisamicheal) [reply src/store/modules/contact.us.js:27]: yeah

**Pull request merged**